faq.html

```diff
@@ -0,0 +1,155 @@
+                    There is no ETA on this, and there will be many bugs and it will take a while until it's ready to begin porting.
+                </h6>
+            </div>
+            <h3>Does Vita3K support commercial games?</h3>
+            <div class="my-5">
+                <h6>
+                    <strong>Some</strong>. The emulator is in an early stage of development and a lot of features required to run many commercial games haven't been implemented yet. Currently, many games made with GameMaker Engine work flawlessly.
+                    More games will be supported when the hardware/firmware emulation is accurate enough to run them. It all depends on the amount of developers working on this project, their skills, and free time, etc. Ignore any estimates and guesses you find out there.
+                </h6>
+            </div>
+            <h3>How to get games?</h3>
+            <div class="my-5">
+                <h6>
+                    For homebrew, get them via <a href="https://vitadb.rinnegatamante.it/">VitaDB</a>
+                    For commercial games, dump them with <a href="https://github.com/TheOfficialFloW/NoNpDrm/releases">NoNpDrm</a>. If the game is known to not work with NoNpDrm, you can optionally replace the executable with a <a href="https://github.com/CelesteBlue-dev/PSVita-RE-tools/tree/master/FAGDec/build">FAGDec</a> dump. Vitamin dumps are <strong>not</strong> supported.
+                </h6>
+            </div>
+            <h3>How to run games?</h3>
+            <div class="my-5">
+                <h6>First you have to install them. When you do, you only need to run the emulator and it will display a <em>Game Select</em> Screen.</h6>
+            </div>
+            <h3>How to install games?</h3>
+            <div class="my-5">
+                <h6>
+                    Windows:<br />
+                    Right click on a <code>.vpk</code>, go to Properties and click "Change..." to associate <code>Vita3K.exe</code> with <code>.vpk</code> files. Double-clicking them should work after that. Just make sure to replace everything in the same path when updating Vita3K, or do the association process again for newer Vita3K builds.
+                    Alternatively, drag and drop games to <code>Vita3K.exe</code>.<br /><br />
+                    Windows/Linux/macOS:<br />
+                    Supply them as a command-line argument to the executable (the latter is necessary for Linux/Mac OS).
+                    Alternatively, install them manually by unzipping the <code>.vpk</code>  to Vita3K's home directory (see below).
+                    Note: If you dumped via MaiDump, the game will not be in a <code>.vpk</code>. Either zip it up (the contents, not the root folder) and (optionally) rename it to <code>.vpk</code>, or install it manually as described above.
```

Comment:
I've been out of the loop for a while but aren't MaiDumped games poorly supported now?
It's also good to mention that games can be installed via the emulator directly through the `File > Install .pkg` or `File > Install .zip .vpk` and this should probably be the recommended way for end-users to install their games since it's easier.